### ISSUE TITLE
Combining large background-image as data-uri

### ISSUE THREAD

**Issue by lupugabriel1** [jakubpawlowicz/clean-css#437]
clean-css does not have a way to treat large background-images defined as data-uris. I'll give you an example of jstree library that was processed to convert resources to data-uris:
```css
 #jstree-dnd.jstree-dnd-responsive > .jstree-ok {
    background-image: url(data:image/png;[----VERY_LARGE_STRING----]=);
    background-position: 0 -200px;
    background-size: 120px 240px;
  }
  #jstree-dnd.jstree-dnd-responsive > .jstree-er {
    background-image: url(data:image/png;[----VERY_LARGE_STRING----]=);
    background-position: -40px -200px;
    background-size: 120px 240px;
  }
```
If cssmin would figure out that ```background-image: url(data:``` should be treated independently and update the code to
```css
 #jstree-dnd.jstree-dnd-responsive > .jstree-ok,
 #jstree-dnd.jstree-dnd-responsive > .jstree-er {
   background-image: url(data:image/png;[----VERY_LARGE_STRING----]=);
}
 #jstree-dnd.jstree-dnd-responsive > .jstree-ok {    
    background-position: 0 -200px;
    background-size: 120px 240px;
  }
  #jstree-dnd.jstree-dnd-responsive > .jstree-er {
    background-position: -40px -200px;
    background-size: 120px 240px;
  }
```
my total size of the css would decrease with 50KB, representing 20% of the total amount of the file. I assume that optimizing this, would dramatically decrease size of files that contain data-uri for different resources ( background-image, fonts ).
Is there any tweak to achieve the above mentioned behavior, or a new processor is required ?

**Comment by jakubpawlowicz** [comment]
@lupugabriel1 It is in my pipeline indeed and should be ready soon-ish!

**Comment by lupugabriel1** [comment]
Cool @jakubpawlowicz, I'll add a gist of a custom solution I developed within a parse method (as a temp solution), maybe it helps. Thanks for your feedback !

**Comment by jakubpawlowicz** [comment]
Sure, please do :+1:

**Comment by lupugabriel1** [comment]
Here it is https://gist.github.com/lupugabriel1/a7e2312480ffd93b971b @jakubpawlowicz . Thanks !

**Comment by jakubpawlowicz** [comment]
Thanks for sharing. It will be a part of advanced optimizer so rather operating on tokenized content instead. Will keep you posted about progress.

**Comment by jakubpawlowicz** [comment]
@lupugabriel1 It's been fixed in 9aa4f49069208950c3090f4c52d2bb2705094a88 on master and in 3.1 soon.
